package.json

```diff
@@ -5,7 +5,6 @@
   "publishConfig": {
     "access": "public"
   },
-  "type": "module",
```

Comment:
@carsonfarmer this is meaningless because we are building `dist/cjs` and `dist/esm`, and `fixup.sh` (which runs as the last thing when running `npm build`) generates `dist/cjs/package.json` and `dist/esm/package.json` which each contain the correct `"type"` declaration.

Reply:
To clarify, we could leave this, but it is meaningless because it is overridden by the dist package.json files. I think leaving this here just creates potential confusion.

Reply:
gotcha

Reply:
@carsonfarmer I may bite my tongue here. That top level `"type"` could be useful for running node scrips (like mentioned above), but also for running tests that are written in typescript.

Reply:
So maybe it is just easier to leave `"type": "module"` in here.

Reply:
Yeh, module is good. I'm going through the same pains in js-tableland right now. This whole build system is broken. It is annoying.